src/components/shared/navbar/Navbar.tsx

```diff
@@ -1,43 +1,34 @@
-import autobind from "autobind-decorator";
 import classNames from "classnames";
 import * as React from "react";
-
 import IconButton from "../icon-button/IconButton";
-
 import "./Navbar.scss";
 
-interface INavbarProps {
+interface NavbarProps {
   backButton?: boolean;
   title?: string;
   onBackButtonClick?: () => void;
   className?: string;
 }
 
-@autobind
-class Navbar extends React.Component<INavbarProps, unknown> {
-  public render() {
-    const { backButton, title, className } = this.props;
-
-    return (
-      <div className={classNames("navbar", className)}>
-        {backButton && (
-          <IconButton
-            className="navbar__back-button"
-            iconName="arrow_back"
-            id="navbar-back-button"
-            onClick={this.onBackButtonClick}
-          />
-        )}
-        {title && <div className="navbar__title">{title}</div>}
-      </div>
-    );
-  }
-
-  private onBackButtonClick() {
-    if (this.props.onBackButtonClick) {
-      this.props.onBackButtonClick();
-    }
-  }
+function Navbar({
+  backButton,
+  title,
+  className,
+  onBackButtonClick,
+}: NavbarProps) {
+  return (
+    <div className={classNames("navbar", className)}>
```

Comment:
Again for accessibility, could this use the `nav` semantic markup or be given the [`navigation` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role)?

Reply:
Yep!